core/src/main/scala/org/bitcoins/core/wallet/builder/TxBuilderError.scala

```diff
@@ -127,15 +139,19 @@ object TxBuilderError {
 
   /**
     * Means that the signed version of this transaction has MORE outputs than what was specified
-    * when building the [[TxBuilder]]. [[TxBuilder.destinations]] && [[TxBuilder.changeOutput]] should
+    * when building the [[org.bitcoins.core.wallet.builder.TxBuilder TxBuilder]].
+    * [[org.bitcoins.core.wallet.builder.TxBuilder.destinations TxBuilder.destinations]] &&
+    * [[org.bitcoins.core.wallet.builder.TxBuilder.changeSPK TxBuilder.changeSPK]] should
```

Comment:
Changed `changeOutput` to `changeSPK`, as `changeOutput` does not exist on `TxBuilder`. Was this correct?

Reply:
Yes, we since we do not know the amount we cannot have a change output.